compiler/ml/printtyp.ml

```diff
@@ -571,8 +571,48 @@ let reset_and_mark_loops_list tyl =
 
 (* Disabled in classic mode when printing an unification error *)
 
+let remove_stdlib t =
```

Comment:
I was wondering what's done for Pervasives, and there's some logic already around `non_shadowed_pervasive`, which could be adapted.

Reply:
In general, I would do a search for "Pervasives", and see where there is logic to be copied.

Reply:
Great lead, will have a look!